src/control/layer/LayerController.cpp

```diff
@@ -222,6 +227,53 @@ void LayerController::moveCurrentLayer(bool up) {
     fireRebuildLayerMenu();
 }
 
+void LayerController::mergeCurrentLayerDown(void) {
+    control->clearSelectionEndText();
+
+    PageRef page = getCurrentPage();
+    auto pageID = selectedPage;
+    if (page == nullptr) {
+        return;
+    }
+
+    // layerID value:
+    //    ...
+    //    2: layer 2
+    //    1: layer 1
+    //    0: background
```

Comment:
Please use block comments for multi-line comments. Using single-line comments for multiple lines can lead to serious problems down the way. (MISRA coding style)
```
/*
 *
 */
```

Reply:
Done, but I don't understand the reasoning yet; can you expand on this?  
From looking at MISRA C++ 2008 I can only find Section 6.2.7, which states nothing like this. I wasn't familiar with MISRA before this, but I did look at Xournalpp's Coding Conventions wiki page, which doesn't state anything about this.